block-styles.css

```diff
@@ -0,0 +1,20 @@
+.wp-block-code textarea {
+	background: transparent;
+}
+
+.code-block-bg {
+	left: 0;
+	position: absolute;
+	top: .8em;
+	width: 100%;
+	z-index: -1;
+}
+
+.code-block-bg .loc {
+	display: block;
+	height: 1.65em;
+}
+
+.code-block-bg .loc.highlighted {
+	background: rgba(86, 213, 255, 0.2);
+}
```

Comment:
This would also need to get added to the frontend inline styles, correct?

Reply:
I'm thinking of having separate styles for the frontend vs the editor. Reason being is the frontend handles highlighting lines differently from what the editor is doing. Where in the editor we need to "style" a `textarea`, the frontend has a lot of dedicated divs we can style.